maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java

```diff
@@ -79,6 +81,8 @@
 @ApplicationScoped
 public class RepositoryManagerDriver implements RepositoryManager {
 
+    private static final String EXTRA_PUBLIC_REPOSITORIES_KEY = "EXTRA_PUBLIC_REPOSITORIES";
```

Comment:
Do we need to use the word PUBLIC here? I think the repos might be inside VPN also (typically a Brew repo), Indy is able to access them.

Reply:
OK, I can drop the word public